custom_components/zaptec/switch.py

```diff
@@ -17,13 +17,13 @@ async def async_setup_platform(
     hass: HomeAssistantType, config: ConfigType, async_add_entities, discovery_info=None
 ) -> bool:  # pylint: disable=unused-argument
     """Setup switch platform."""
-    api = hass.data.get(DOMAIN, {}).get("api")
-    if api is None:
+    acc = hass.data[DOMAIN]["api"]
```

Comment:
Use .get like I did before. This will raise and exception and not return None

Reply:
fixed